contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -1,63 +1,284 @@
 var Backbone = require("backbone");
 var _= require("underscore");
 
-window.ChannelEditDataModel = Backbone.Model.extend({
-	urlRoot: '/channel_edit',
-	initialize: function() {
-		//_.bindAll(this);
-	}
-});
-
-/* TODO: CHANGE TO ChannelEditDataModel.EXTEND */
-var TopicNodeModel = Backbone.Model.extend({
+var NodeModel = Backbone.Model.extend({
 	defaults: {
-        color1:"",
-		color2:"",
-		color3:"",
 		title:"Untitled",
-		description:"This is the description generated by default"
-    }
+		description:"No description",
+		parent: null,
+		children:[],
+    },
+    urlRoot: function() {
+		return window.Urls["node-list"]();
+	},
+	toJSON: function() {
+	  var json = Backbone.Model.prototype.toJSON.apply(this, arguments);
+	  json.cid = this.cid;
+	  return json;
+	},
+	parse: function(response) {
+	    return _(response).isArray()
+	         ? response[0]
+	         : response;
+	},
+
+	/*Used when copying items to clipboard*/
+    duplicate: function(parent_id){
+    	var title = this.generate_title(this.get("title"));
+		var data = {
+			title: title,
+			created : this.get("created"),
```

Comment:
Consider using the ```pick``` method to get a list of the keys that you want, and then extending the data object with the additional ```title``` and ```parent``` attributes.